docs/faq.md

```diff
@@ -31,16 +31,7 @@ https://github.com/docker/docker/blob/master/LICENSE)
 
 ### Does Docker run on Mac OS X or Windows?
 
-Docker Engine currently runs only on Linux, but you can use VirtualBox to run
-Engine in a virtual machine on your box, and get the best of both worlds. Check
```

Comment:
re-reading the original text, we can keep both as just vBox is complete and so is Docker for Mac / Windows as it depends on their machine